uniref-rest/src/main/java/org/uniprot/api/uniref/controller/UniRefEntryController.java

```diff
@@ -97,18 +96,15 @@ public UniRefEntryController(
                             flags = {Pattern.Flag.CASE_INSENSITIVE},
                             message = "{search.invalid.id.value}")
                     String id,
```

Comment:
I think, `id` is not telling much. We should use some other name like "clusterId" or "unirefClusterId". 
What do you say?

Reply:
Jie used id (we are using id for many others as well... I  do not have problems with both... this is the path parameter name as well